pyvo/dal/vosi.py

```diff
@@ -14,10 +14,10 @@
 from ..utils.decorators import stream_decode_content, response_decode_content
 from ..utils.http import use_session
 
-__all__ = ['CapabilityMixin', 'VOSITables']
+__all__ = ['AvailabilityMixin', 'CapabilityMixin', 'VOSITables']
```

Comment:
AvailabilityMixin is deprecated, please don't put it back here.

Reply:
Oh I missed that it was deprecated, I've removed it again